exercises/J-array-get-set/exercise.js

```diff
@@ -5,11 +5,13 @@
 */
 
 function first(arr) {
-  return; // complete this statement
+  return numbers[0]; { 
```

Comment:
The first return value is correct as you are returning the first index of the array.The second return statement should be removed